config.json.in

```diff
@@ -46,5 +46,35 @@
       "retries": 2,
       "minTimeout": 500
     }
+  },
```

Comment:
This change is unnecessary (config.json.in) as this file is no longer used. I'll be removing it as part of #13. The config is now generated by config-agent using the sapi_manifests/vmapi/template file.